### ISSUE TITLE
IE Blank Admin  (JSON parsing issue 'List View' props)

### ISSUE THREAD

**Issue by cantuket** [keystonejs/keystone#4769]
### Expected behavior

Admin should be usable in IE 11

### Actual/Current behavior

- Any 'List' view in the Admin shows a blank 'loading' screen indefinitely

![keystone4-ie-error](https://user-images.githubusercontent.com/4649849/44119094-d1e71ace-9fdd-11e8-9836-df3a44e38fec.png)

- IE throws error originating from...
showBlankState() 
admin/client/App/screens/List/index.js:397  

![screenshot 7](https://user-images.githubusercontent.com/4649849/44120605-8255e2f6-9fe2-11e8-9dac-06a89c93b9ec.png)

As you can see above the 'items' prop on the ListView component is coming in as string of JSON.

I'm not sure where you'd like to handle this best, but it seems like adding a JSON.parse() in the response from loadItems() action's helper function itemsLoaded() would be a good spot, but unfortunately I can't seem to to get keystone to compile a new bundle so I can't test.

admin/client/App/screens/List/actions/items.js:90
```
export function itemsLoaded (items) {
	return {
		type: ITEMS_LOADED,
		items: JSON.parse(items),
	};
}
``` 

But, the API appears to be sending back a properly formatted response so error could be somewhere further downstream. I'll try and figure out how to keystone compiled and test, but until then hopefully someone else can submit a fix.

### Steps to reproduce the actual/current behavior

NA

### Environment

| Software         | Version
| ---------------- | -------
| Keystone         |  4.0.0 
| Node.js          |   9.2.1
| Browser          |  IE 11

**Comment by stennie** [comment]
@cantuket Possibly the same issue as #3734? There's a potential fix in PR #4641 but there was debate on whether this was the correct fix.

Will ask for some volunteers to help progress this.

Regards,
Stennie

**Comment by cantuket** [comment]
@stennie Oops, sorry you're right. Not sure how I didn't find those existing issues.

I'll close this in favor of https://github.com/keystonejs/keystone/pull/4641 and leave my thoughts there especially because this should be handled in the queries not the actions.
